server/apps/api/management/commands/update_ip_data.py

```diff
@@ -1,12 +1,102 @@
 # -*- coding: utf-8 -*-
 
+import json
+import hashlib
+from time import sleep
+
+import requests
+
 from django.core.management.base import BaseCommand
+from django.db.models import Count
+
+from server.apps.api.models import Case, Domain
+from server.apps.api.logic import notifier
 
-from server.apps.api.logic.ips_data import update_ip_data
-from server.apps.api.logic.blocked import check_blocked
+
+MIN_IP_COUNT = 2
```

Comment:
It's hard to understand what this means? What if there are only 2 IPs?

Reply:
Im having trouble with short and clear name for this constant. Could you give some advice, please? It's a treshold, a minimum number of cases from different IPs to consider a domain blocked.

Reply:
`MIN_CASE_COUNT_PER_DOMAIN` or something like that.